serverless/serverless
# Listing stacks

## What did you implement:

Closes #2410

## How did you implement it:

Fetching the S3 Objects from the deployment folder based on the default stage & region. In case no entries are found the following message is shown:

```
Couldn\'t find any existing stacks.
Please verify that stage and region are correct.
```

This uses the same findAndGroupStacks helper functions as in this PR https://github.com/serverless/serverless/pull/2436. Since it's exactly the same file it shouldn't even create a merge conflict.

## Preview

![screen shot 2016-10-18 at 15 41 32](https://cloud.githubusercontent.com/assets/223045/19480531/f8003c12-954a-11e6-8e6c-709f6aa8fb31.png)

## How can we verify it:

1. Deploy a service and run `serverless deploy list`
2. You should see your deployment listed.

## Questions

@flomotlik I believe it would be useful in case `deploy list` would accept `stage` and `region` as parameters. If yes, should I add it in this PR or another?
Please also check the preview if it the output is consistent with other commands.

***Is this ready for review?:*** YES
